src/bin/restore.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env babel-node
+
+const parser = require('../cdoFlavoredParser');
+const parseArgs = require('minimist')
+const fs = require('fs');
+
+const argv = parseArgs(process.argv.slice(2));
+
+const sourcefile = argv.s
+let sourcedata = fs.readFileSync(sourcefile);
```

Comment:
nit: these should be camelCase, e.g. sourceData, sourceFile, redactedData, redactedFile. Since these are new files, now might be a good time to change them, even if you don't go back and change this in other places.